types/jss/jss-tests.ts

```diff
@@ -7,10 +7,11 @@ import {
 
 const jss = createJSS().setup({});
 
-const styleSheet = jss.createStyleSheet(
+// use <string> because we will be dynamically augmenting
+const styleSheet = jss.createStyleSheet<string>(
 	{
-		ruleWithMockObservable: {
-			subscribe() {}
```

Comment:
Was this test a mistake? I don't understand how it makes sense to pass a "mock observable" as a `Style`.

Reply:
@kof was working on some observable API. Maybe it's related.

Reply:
Indeed, JSS accepts observables as style values.  Please revert this.

Reply:
Ok. Observables of what?

Reply:
In [this example](https://github.com/cssinjs/examples/blob/gh-pages/observables/index.js) it looks like observables can be used in place of individual property values. But this test is saying that entire rules can be observed, is that correct?

Reply:
Yeah, you can do this:

```typescript
const ballStyle$ = new Subject();

const sheet = jss.createStyleSheet(
  {
    container: {
      touchAction: 'none',
      overflow: 'hidden',
    },
    ball: ballStyle$,
  },
  {
    link: true,
   }
).attach();

ballStyle$.next({
  'background-color': 'red',
  width: '100px',
  height: '100px',
});
```

For example, https://github.com/material-motion/material-motion-js/blob/develop/packages/demos-react/src/JSSDemo.tsx#L54